python/ray/tests/test_advanced_8.py

```diff
@@ -192,6 +192,18 @@ def test_ray_resources_environment_variable(ray_start_cluster):
     assert cluster_resources["CPU"] == 3
 
 
+def test_ray_labels_environment_variables(shutdown_only):
+    os.environ[
+        ray_constants.LABELS_ENVIRONMENT_VARIABLE
+    ] = '{"custom1":"1", "custom2":"2"}'
+    ray.init(labels={"custom1": "3", "custom3": "3"})
```

Comment:
Hmm should we support labels from `ray.init`? I am not sure if this was already discussed, but it seems not useful. If it is a single node, ability to pass label is not necessary?

Reply:
Yea, you are right it's not useful in practice but it does make things symmetric (you can set node labels in all ways of starting a node). After all, users can always use `ray start` to start a single node cluster with node labels.

It's also convenient for users to try labels out.